core/app/models/spree/shipment.rb

```diff
@@ -33,7 +33,10 @@ class Shipment < Spree::Base
     scope :trackable, -> { where("tracking IS NOT NULL AND tracking != ''") }
     scope :with_state, ->(*s) { where(state: s) }
     # sort by most recent shipped_at, falling back to created_at. add "id desc" to make specs that involve this scope more deterministic.
-    scope :reverse_chronological, -> { order('coalesce(spree_shipments.shipped_at, spree_shipments.created_at) desc', id: :desc) }
+    scope :reverse_chronological, -> {
+      order(Arel.sql('coalesce(spree_shipments.shipped_at, spree_shipments.created_at) desc'), id: :desc)
```

Comment:
I think we can take advantage of this change to also change `Spree::Shipment.table_name` instead of the hardcoded `spree_shipments`

Reply:
You mean something like `#{Spree::Shipment.table_name}.shipped_at`, right? That would help in case we were to change the table's name, is that correct?

Reply:
Yes, exactly.

Reply:
Sounds reasonable, I'll push a new commit for it :smile_cat: